spec/spec_helper.cr

```diff
@@ -1,10 +1,28 @@
 require "digest"
-require "spec"
+require "spectator"
+require "spectator/should"
 
 require "../src/pluto"
 
 module SpecHelper
   def self.read_sample(name : String) : String
     File.read("lib/pluto_samples/#{name}")
   end
+
+  def new_rgba_pluto : Pluto::RGBAImage
+    Pluto::RGBAImage.from_ppm(File.read("lib/pluto_samples/pluto.ppm"))
+  end
+
+  def new_gray_pluto : Pluto::GrayscaleImage
+    Pluto::GrayscaleImage.from_ppm(File.read("lib/pluto_samples/pluto.ppm"))
+  end
+
+  macro with_images
+    let(image) { new_rgba_pluto }
+    let(gray_image) { new_gray_pluto }
+  end
```

Comment:
Not used in example above, but we can also use macros to pre-define some `let` variables directly. Macros could further be used to write the specs, where the macro is essentially given a method name to run and a set of digests to compare with, it could generate both the 'RGBAImage' and 'GrayscaleImage' spec versions of it. The last thought felt like too much to do all at once and I wanted to get opinions first, so not done here either.

Reply:
Quite like this `with_images` approach